examples/simple_threshold_signature/threshold_signature.rs

```diff
@@ -0,0 +1,89 @@
+use crate::simple_threshold_signature::signature::Signature;
+use crate::Element;
+use alba::centralized_telescope::proof::Proof;
+use alba::centralized_telescope::Telescope;
+use blst::min_sig::{PublicKey, Signature as BLSSignature};
+use blst::BLST_ERROR;
+
+pub(crate) struct ThresholdSignature {
+    proof: Proof,
+}
+
+impl ThresholdSignature {
+    pub(crate) fn aggregate(
+        signatures: &[Signature],
+        alba: &Telescope,
+        key_list: &[(usize, PublicKey)],
+    ) -> (Self, Vec<PublicKey>) {
+        let prover_set = signatures
+            .iter()
+            .map(|s| s.signature.to_bytes())
+            .collect::<Vec<Element>>();
+        println!("-- Creating alba proof. ");
+        let proof = alba.prove(&prover_set).unwrap();
+        println!("-- Alba proof created: ");
+        println!(
+            " - Numbers of retries done to find the proof: {}",
+            proof.retry_counter
```

Comment:
Should we implement getters for the proof and other structs instead of having public fields (like for Telescope)? (We would do it in another PR of course)

Reply:
Let's do it in another PR 👍